packages/ember-runtime/lib/mixins/-proxy.js

```diff
@@ -83,6 +80,10 @@ export default Mixin.create({
   */
   content: null,
 
+  _initializeProxy: on('init', function() {
+    meta(this).setProxy();
```

Comment:
Ahh, actually the tag initialization needs to stay as an on('init') still, this is because content can be set in init (after _super is called by folks extending). It's "special" here (maybe it warrants a comment to make it clearer for future travelers)?